//package core.io.protocol;

message DseStageAward
{
	optional int32	type			= 1;//1普通2精英
	optional int32 	chapterid	 	= 2;//章节ID
	optional int32	sceneid			= 3;//场景ID
	optional int32	res				= 4;//结果
}
